workbench-core/example/infrastructure/integration-tests/support/utils/settings.ts

```diff
@@ -13,6 +13,8 @@ export interface Setting {
   rootUserNameParamStorePath: string;
   rootPasswordParamStorePath: string;
 
+  rootUserId: string;
```

Comment:
don't recognize this change

Reply:
you can revert this. It was moved down to line 35 (or 37)